engine/src/main/java/io/zeebe/engine/state/message/MessageState.java

```diff
@@ -319,6 +319,16 @@ public void remove(final long key) {
         }));
   }
 
+  public boolean isEmpty() {
```

Comment:
Yes. The `isEmtpy()` methods are used in tests only. I think it's ok because it is simple and doesn't expose some internals of the class. I see no other ways to verify that the state is emtpy.